vscode-extension/package.json

```diff
@@ -34,6 +34,11 @@
           "default": true,
           "description": "Enable formatting with scalafmt"
         },
+        "scalameta.scalafmt.onSave": {
+          "type": "boolean",
+          "default": false,
+          "description": "Format file on save"
```

Comment:
How is this different from `  "editor.formatOnSave": false`? I guess that's vscode specific and this LSP?

Reply:
Yes, `editor.formatOnSave` is VSCode specific. Also I think it formats after saving the file. Probably this needs a clarification in the description.